### ISSUE TITLE
vibranceGUI isn't working with cloned displays.

### ISSUE THREAD

**Issue by tatne** [juvlarN/vibranceGUI#13]
Someone had issue with this program not working and only solution was to disconnect HDMI from TV.
I have this same issue and I have 1 monitor and its picture cloned to my TV via HDMI cable.
Is it possible to have this setup working without always unplugging HDMI cable?

**Comment by juvlarN** [comment]
Hi,

for a quick fix/bypass, try pressing Windows+P and only select your main monitor before you are gaming. Might have to restart vibranceGUI, not sure.

The current code should only affect your main monitor (the one where csgo is running on). What's the problem you are encountering? Does vibranceGUI completely stop working? 

Thanks

**Comment by tatne** [comment]
Thanks for the tip!

My main display is BenQ 21,5" monitor and I've cloned it to my TV from nvidia control panel. So CSGO is running on both of those displays simultaneously. vibranceGUI doesn't work AT ALL, not in Windows and it doesn't react to games also.

**Comment by juvlarN** [comment]
this is weird. because it is working with 2 monitors without any problem for me.
Unfortunately I dont have a tv that I could plug into my computer. Not sure what's causing the problem.

Does the bypass work for you? So we might have narrowed it down to a TV being plugged into the graphics card.

**Comment by tatne** [comment]
I discovered something weird with Win+P. If I select 'PC Screen only' my monitor loses signal and only TV shows picture. So my Windows treats TV as primary monitor. And when I select this option and launch CSGO the program woks and TV changes vibrance. When I select 'Second' screen only' TV loses signal and monitor comes back alive. But with this setting automatic vibrance changing isn't working.

So I guess it has something to do with that. I've made my monitor the primary display from nvidia cp but for some reason Windows still thinks TV is my main display. I've connected my monitor with normal DVI-D cable and my TV with HDMI to DVI-D cable. I have GTX 970 graphics card. I'll test if changing the dvi ports other way around has effect.
